domains/tts.deva.is-cool.dev.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "ttsapi",
+
+    "domain": "is-cool.dev",
+    "subdomain": "tts.deva",
+
+    "owner": {
+        "repo": "https://github.com/turbomaster95/turbomaster95.github.io",
+        "email": "devamidhun.3c.kmbvm@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "48c10554-32ea-4bc5-98f8-5f062b511ad1.id.repl.co"
```

Comment:
CNAME displays no content.

Reply:
oh yeah replace that with 48c10554-32ea-4bc5-98f8-5f062b511ad1.id.repl.co

Reply:
That's also invalid.

Reply:
yeah it is running now pls merge

Reply:
use the url : 48c10554-32ea-4bc5-98f8-5f062b511ad1.id.repl.co
it is now running

Reply:
> 48c10554-32ea-4bc5-98f8-5f062b511ad1.id.repl.co

Not loading.

Reply:
It is replit if I close the tab running replit I shuts down if u pls merge this and the domain becomes public then it won't shut down